python_modules/libraries/dagster-dbt/dagster_dbt/__init__.py

```diff
@@ -141,6 +144,33 @@
             ("dbt_test_op", "dagster_dbt.ops"),
         ]
     },
+    **{
+        value: (
+            module,
+            "0.24.0",
+            f"Use the @dbt_assets decorator, DbtCliResource, and DagsterDbtTranslator instead. {additional_text}",
+        )
+        for value, module, additional_text in [
+            (
+                "load_assets_from_dbt_manifest",
+                "dagster_dbt.asset_defs",
+                (
+                    "To generate a dbt manifest at run time in your Python code using `dbt parse`,"
+                    " see the reference:"
+                    " https://docs.dagster.io/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project.",
+                ),
+            ),
+            (
+                "load_assets_from_dbt_project",
```

Comment:
I would have expected https://docs.dagster.io/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project to accompany `load_assets_from_dbt_project`, no?